apps/fz_wall/lib/fz_wall/cli/live.ex

```diff
@@ -6,104 +6,129 @@ defmodule FzWall.CLI.Live do
   specified IP addresses, ports, and protocols from Firezone device IPs.
   """
 
+  import FzWall.CLI.Helpers.Sets
+  import FzWall.CLI.Helpers.Nft
```

Comment:
Elixir imports are lexically scoped and can be considered syntax sugar (making functions available without a fully qualified module name, or its alias).

`teardown_table` is missing hence causing the error.

Reply:
My bad, made some refactoring that seemed trivial at the moment and moved some additional things to `Nft` and didn't test that with the live version.